docs/readme.md

```diff
@@ -156,4 +156,12 @@ If the underlying data source does not allow such a fine-grained distinction, us
 		currency: 'EUR' // ISO 4217 code, required
 	}
 }
-```
\ No newline at end of file
+```
+
+## modes
+
+- `walking`
+- `train`
+- `bus` - local traffic
+- `coach` - long-distance traffic
```

Comment:
Ok, I'm fine with that. Especially since it would be inconsequent to have different modes for buses but not for trains. But that's where things would get **really** messy, so lets keep the modes as broad as possible.